### ISSUE TITLE
Unable to create an order via Magento Admin after upgrading to version 2.4.6

### ISSUE THREAD

**Issue by RobertSokolovSW** [magento/magento2#37454]
### Preconditions and environment

- Magento version 2.4.6 Enterprise Edition

### Steps to reproduce

1. 2.4-develop with Enterprise edition modules enabled.
2. Setup multi store environment as per below screenshot and as per [docs](https://devdocs.magento.com/guides/v2.3/config-guide/multi-site/ms_websites.html)
![image](https://github.com/magento/magento2/assets/60198231/d01b753a-a2f4-4dc8-bc2d-17fc85bc358a)
Note: Make sure to setup more than 2 websites.
4. Create new customer account from front-end
5. Create new store view again from Stores - All stores - Create store view and assign to main website.
6. Re index and clear cache
7. Sales - Orders - Create new order - select customer and select new store view , select product and submit order

### Expected result

The Create Order Page is loaded without any errors.

### Actual result
**Issue**: Unable to place order.
[2023-05-15T09:42:27.771015+00:00] report.ERROR: Unable to resolve the source file for 'adminhtml/Magento/spectrum/en_US/moment.min.js.map' [] []
[2023-05-15T09:42:27.771443+00:00] report.CRITICAL: Unable to resolve the source file for 'adminhtml/Magento/spectrum/en_US/moment.min.js.map' [] []
![image](https://github.com/magento/magento2/assets/60198231/03a6da0f-51db-4efe-b989-dd4130e30e28)

### Additional information

[Screenshot](https://prnt.sc/gNFOB8gssOfw)

Clicking "Create new customer" on the customer selection page, has a different behavior. After clicking this button and then picking a store view on the next page, you are taken to the Create Order page, but then quickly redirected back to the Store View selection page.

### Triage and priority

- [ ] Severity: **S0** _- Affects critical data or functionality and leaves users without workaround._
- [X] Severity: **S1** _- Affects critical data or functionality and forces users to employ a workaround._
- [ ] Severity: **S2** _- Affects non-critical data or functionality and forces users to employ a workaround._
- [ ] Severity: **S3** _- Affects non-critical data or functionality and does not force users to employ a workaround._
- [ ] Severity: **S4** _- Affects aesthetics, professional look and feel, “quality” or “usability”._

**Comment by RobertSokolovSW** [comment]
> #37452 Looks like the same issue

This issue has different reproduction steps and different output, but yeah, both issues seem related.

**Comment by Yaroslav1301** [comment]
I'm having the same issue on my newly updated Magento instance 2.4.6. and has temporarily helped this fix https://github.com/magento/magento2/issues/37452#issuecomment-1533497213

**Comment by engcom-November** [comment]
@magento give me 2.4-develop instance with edition ee

**Comment by engcom-November** [comment]
Verified the issue on Magento 2.4-develop instance with enterprise edition modules enabled and the issue is reproducible with below steps performed:
**Steps performed:**
1. 2.4-develop with Enterprise edition modules enabled.
2. Setup multi store environment as per below screenshot and as per [docs](https://devdocs.magento.com/guides/v2.3/config-guide/multi-site/ms_websites.html)
3. Create new customer account from front-end
4. Create new store view again from Stores - All stores - Create store view and assign to main website.
5. Re index and clear cache
6. Sales - Orders - Create new order - select customer and select new store view , select product and submit order
![image](https://github.com/magento/magento2/assets/60198231/0471b2cb-4b0f-4c82-9ce3-3d7b6e94a923)
Issue: Unable to place order.
[2023-05-15T09:42:27.771015+00:00] report.ERROR: Unable to resolve the source file for 'adminhtml/Magento/spectrum/en_US/moment.min.js.map' [] []
[2023-05-15T09:42:27.771443+00:00] report.CRITICAL: Unable to resolve the source file for 'adminhtml/Magento/spectrum/en_US/moment.min.js.map' [] []

**Comment by zepgram** [comment]
Issue #37452 describe exactly the same problem and has been closed by @engcom-Dash 
![image](https://github.com/magento/magento2/assets/16258478/80e61db3-5726-410a-b05a-29257fc0b9e7)

This a major breaking change for retailers, we are talking about taking orders from back-office, this should never happen in real life, and when community report issue, you just close it ? We are not talking about a wishlist or catalog issue or even customer area, this is the back-office checkout, one of the most critical point for retailers especially on B2B.

Where is the quality here ? This is not the first time. When the community report a bug, could you please take it in consideration and double check before closing tickets ?

Every release we got new bugs, I got this now, but please when you launch a release and people report critical bug like this, take it in consideration and provide a patch to fix it.

**Comment by chittima** [comment]
Internal team fixed the issue and here is the commit id: https://github.com/magento/magento2/commit/d6176449f759a8bd535693cdf2971aed5a14e4c3

**Comment by ElleGee157** [comment]
When trying to create an order in Magento 2.4.4 you press Add products, nothing is added and the add product button disappears. Please help.

**Comment by engcom-November** [comment]
Hello @RobertSokolovSW,

As we are in the process of fix this issue, we tried to reproduce this issue in multiple Mac OS machines but the issue is not reproducible for us.
Can you please let us know the OS and it's version in which you are able to reproduce this issue.

Thanks.

**Comment by RobertSokolovSW** [comment]
Hello @engcom-November 

The issue was reproduced on Ubuntu 20.04.5 LTS.

Let me know if there is anything else you need.

**Comment by ElleGee157** [comment]
Hello @engcom-November 

The issue was reproduced on Windows 11 Home Version	22H2

**Comment by Prasanna-Adobe** [comment]
@magento give me 2.4.6 instance with edition ee

**Comment by amcguireweb** [comment]
This issue persists for me on 2.4.7-p1, and I have confirmed that the PR changes from this fix is present in **vendor/magento/module-sales/view/adminhtml/web/order/create/scripts.js**.

**Comment by jmonrove** [comment]
Still an issue on 2.4.6-p8

**Comment by coresh** [comment]
Confirmed the issue for the v2.4.7

```
Unable to resolve the source file for 'adminhtml/Magento/spectrum/en_US/requirejs.js' 
```

Thank you

**Comment by engcom-Bravo** [comment]
Hi,

We have reverified the issue in Latest 2.4-develop instance with EE edition and we are not able to reproduce the issue.Kindly refer the screenshots.


**Steps performed**:

- 2.4-develop with Enterprise edition modules enabled.
- Setup multi store environment as per below screenshot and as per [docs](https://devdocs.magento.com/guides/v2.3/config-guide/multi-site/ms_websites.html)
- Create new customer account from front-end
- Create new store view again from Stores - All stores - Create store view and assign to main website.
- Re index and clear cache
- Sales - Orders - Create new order - select customer and select new store view , select product and submit order


<img width="1406" alt="Screenshot 2024-12-20 at 4 12 38 pm" src="https://github.com/user-attachments/assets/7f79607f-e985-45dc-9b7b-cc8695edadf9" />



Create Order page is loaded without any errors and able to place the order successfully.

Thanks.

**Comment by hewgim** [comment]
Still an issue in 2.4.6-p10. Is the fix ever going to be released in the 2.4.6 branch ?
